src/main/java/cz/smarteon/loxone/message/ApiInfo.java

```diff
@@ -31,6 +41,8 @@ public String getVersion() {
         return version;
     }
 
+    public Boolean getEventSlots() {return eventSlots;}
```

Comment:
If we know the meaning, would be nice to javadoc it

Reply:
Don't know the meaning